models/qualtrics__contact.sql

```diff
@@ -40,7 +40,7 @@ distribution_response as (
         survey_response.recorded_date
 
     from distribution_contact 
-    join distribution 
+    join distribution
```

Comment:
Same comment about explicitly defining the type of join.

Reply:
updated to left